beacon-chain/rpc/validator/assignments.go

```diff
@@ -37,7 +38,8 @@ func (vs *Server) StreamDuties(req *ethpb.DutiesRequest, stream ethpb.BeaconNode
 	}
 
 	// We compute duties the very first time the endpoint is called.
-	res, err := vs.duties(stream.Context(), req)
+	currentEpoch := slotutil.EpochsSinceGenesis(vs.GenesisTimeFetcher.GenesisTime())
```

Comment:
Given duties are now a server-side stream, the beacon node needs to use an epoch ticker using roughtime to determine the duties instead of expecting an epoch argument in the grpc request